lib/os_places_api/client.rb

```diff
@@ -76,17 +76,38 @@ def any_locations?(results)
     end
 
     def build_locations(results)
+      return build_locations_from_ons(results) if ons_results_only?(results)
+
       locations = unfiltered_locations(results)
 
       raise NoResultsForPostcode if locations.empty?
 
       {
+        "source" => "Ordnance Survey",
+        "retired" => false,
```

Comment:
We reference 'active' in the ADR but 'retired' here?

Reply:
Good spot - to simplify this conceptually I'm going to make it retired everywhere.